### DISCUSSION TITLE
Hide token endpoints on Swagger (from NinjaJWTDefaultController)

### DISCUSSION THREAD

**Discussion by jacinyan** [eadwinCode/django-ninja-extra#191] [Q&A] [answered]
Hi there,

I was wondering if there's a (neat) way of hiding the pair endpoint but keep the remaining two.

Cheers

**Comment by eadwinCode** [comment]
There is but I don't understand what you mean by pair endpoint

**Reply by jacinyan** [reply]
Thank you for your response. Let me clarify my question:

First, I realize now that my question is specifically about django-ninja-jwt, not django-ninja-extra. I apologize for posting in the wrong repository.

In django-ninja-jwt, there are typically three endpoints provided by NinjaJWTDefaultController:

Token creation (often called 'obtain' or 'login')
Token refresh
Token verify

When I mentioned the 'pair endpoint', I was referring to the token creation endpoint.
My question is: Is there a clean way to simply visually hide or disable just the token creation endpoint while keeping the refresh and verify endpoints active in the Swagger UI?

I hope this clarifies my question.

**Reply by eadwinCode** [reply]
okay, I just noticed that the refresh endpoint for `TokenObtainPairController` and `TokenObtainSlidingController` was not detached from the controller into another class. I am afraid you can't do this with the current version of ninja-jwt.

**Reply by eadwinCode** [reply, marked as answer]
I will push up a fix to allow this flexibility
